ray-operator/Makefile

```diff
@@ -92,6 +92,9 @@ docker-build: test docker-image ## Build image with the manager.
 docker-push: ## Push image with the manager.
 	${ENGINE} push ${IMG}
 
+docker-multi-arch-image: # Build and push multi arch image, amd64 and arm64 currently
+	${ENGINE} buildx build --push --platform linux/amd64,linux/arm64/v8 -t ${IMG} .
```

Comment:
Q1: Why do we add `--push` here? 

Q2: What's the difference between `linux/arm64/v8` and `linux/arm64`? In my understanding, they seem to be synonymous. Is my understanding correct?

Reply:
A1: Because this is a multi-arch image, we can not use `--load` here (which will report error), the options left are `--push` or nothing (in which case the built artifact remains in volume of buildx container), so here `--push` is more appropriate.

A2: Actually they are same, reference: https://stackoverflow.com/questions/70819028/relation-between-linux-arm64-and-linux-arm64-v8-are-these-aliases-for-each-othe I put it here for clarity, but if it is ambiguous, we can remove it.

Reply:
Does `--load` mean that load the resulting image into the local Docker daemon's image store? If so, it makes sense to add `--push`.

Reply:
Yes,  `--load` is the meaning you mentioned.

Reply:
Thanks!